schema.go

```diff
@@ -42,8 +42,8 @@ func BlockSchema(name string, v interface{}) (*AST, error) {
 }
 
 // MustBlockSchema reflects a block schema from a Go struct, panicking if an error occurs.
-func MustBlockSchema(name string, v interface{}) *AST {
-	ast, err := BlockSchema(name, v)
+func MustBlockSchema(name string, v interface{}, opt *MarshalOptions) *AST {
```

Comment:
This should be `options...MarshalOption`.

Reply:
Fixed